types/codemirror/index.d.ts

```diff
@@ -1888,4 +1895,32 @@ declare namespace CodeMirror {
             setShowDifferences(showDifferences: boolean): void;
         }
     }
+
+    interface MouseSelectionConfiguration {
```

Comment:
I think all the properties are optional so they should be something like `unit?: ..., extend?: ...`

Reply:
That's true, thank you!